carRental/src/components/CheckBox/CheckBox.jsx

```diff
@@ -1,17 +1,15 @@
 import React from "react";
-
+import "./CheckBox.css";
```

Comment:
Changed all of these class names from form-whatever to checkbox-{whatever}

Reply:
done